projects/contacts-admin/docker-compose.yml

```diff
@@ -29,3 +29,22 @@ services:
       TEST_DATABASE_URL: "mysql2://root:root@mysql-5.5/contacts_test"
       REDIS_URL: redis://redis
 
+  contacts-admin-app: &contacts-admin-app
+    <<: *contacts-admin
+    depends_on:
+      - mysql-5.5
+      - redis
+      - publishing-api-app
+      - nginx-proxy
+      - content-store-app
+      - whitehall-app
+    environment:
+      DATABASE_URL: "mysql2://root:root@mysql-5.5/contacts_development"
+      TEST_DATABASE_URL: "mysql2://root:root@mysql-5.5/contacts_test"
+      REDIS_URL: redis://redis
+      VIRTUAL_HOST: contacts-admin.dev.gov.uk
```

Comment:
I'm a bit late to the party, but trying this out locally breaks due to [this issue](https://github.com/alphagov/govuk-docker/issues/176). Do we need a separate PR to fix that?

Reply:
That PR is here: https://github.com/alphagov/contacts-admin/pull/838

Reply:
Ah, I see - I didn't clock that it was hidden in there, thanks!